frontend/src/layouts/Public.tsx

```diff
@@ -4,12 +4,15 @@ import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 import { faWindowClose } from "@fortawesome/free-solid-svg-icons";
 import EnvConfig from "../services/EnvConfig";
 import logo from "../logo.svg";
+import { useSettings } from "../hooks";
 
 interface LayoutProps {
   children?: ReactNode;
 }
 
 function PublicLayout(props: LayoutProps) {
+  const { settings } = useSettings();
```

Comment:
Since this is a public screen, this should use the `usePublicSettings` hook instead. Otherwise settings won't load for non-authenticated users.